config/lark-parser_xenial.yaml

```
@@ -0,0 +1,12 @@
+name: lark-parser_xenial
+method: http://packages.osrfoundation.org/gazebo/ubuntu-prerelease
+suites: [xenial]
+component: main
+# python packages only, all arches supported
+architectures: [amd64, source]
```

```suggestion
architectures: [amd64, arm64, armhf source]
```

I'm pretty sure the binary package is Architecture: all which means we can import it into all our supported architectures.